S50yolo

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+cat /proc/version
+reboot
```

Comment:
If we just want to shutdown the machine, this could be `poweroff` and just remove the `-no-reboot` QEMU flag but I haven't actually tested yet.

Reply:
That's a good idea. I was thinking of adding that last night but was simply too tired.  Will try it out.

Reply:
It does work, added in https://github.com/ClangBuiltLinux/continuous-integration/commit/aeea4e2b3c3e85adcee912ebdd8182fc6dffeb5f